src/main.js

```diff
@@ -37,23 +35,29 @@ const routes = [
         path: "/",
         component: Tap,
         meta: { auth: true },
-        beforeEnter: function(to, from, next) {
+        beforeEnter: function (to, from, next) {
             if (Store.trader.hasOwnProperty("featureOverride")) {
                 const { tap } = Store.trader.featureOverride.pageAccess;
                 if (tap === false) {
-                    next({path: "/scan"});
+                    next({ path: "/scan" });
                 } else {
                     next();
                 }
             } else {
                 next();
             }
-        }
+        },
     },
     { path: "/account", component: Account, meta: { auth: true } },
     { path: "/scan", component: Scan, meta: { auth: true } },
     { path: "/payment", component: Payment, meta: { auth: true } },
-    { path: "/login", component: Login, meta: { auth: false } },
+    {
+        path: "/login",
+        name: "login",
```

Comment:
The others don't have names. Does it matter?

Reply:
WE call them by path elsewhere; oddly we need a name to do this, apparently 🤷